<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VrcftSettings</class>
 <widget class="QMainWindow" name="VrcftSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRCFT Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="ip_lb">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_le">
      <property name="text">
       <string>localhost</string>
      </property>
      <property name="maxLength">
       <number>256</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="port_lb">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="port_sp">
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>54321</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="read_timeout_lb">
      <property name="text">
       <string>Vrcft Read Timeout (Ms):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="read_timeout_sp">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>60000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="bypass_cb">
      <property name="text">
       <string>Bypass other vrcft module block</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="text">
         <string>Apply and Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
